sounds.json

```diff
@@ -1353,29 +1356,38 @@
     "bedrock_mapping": "PLACE",
     "identifier": "minecraft:copper_grate[waterlogged=false]"
   },
-  "block.copper_grate.hit": {},
-  "block.copper_grate.fall": {},
-  "block.copper_trapdoor.close": {},
-  "block.copper_trapdoor.open": {},
+  "block.copper_grate.hit": {
+    "playsound_mapping": "step.copper_grate",
+    "bedrock_mapping": "HIT"
+  },
+  "block.copper_grate.fall": {
+    "playsound_mapping": "step.copper_grate"
+  },
+  "block.copper_trapdoor.close": {
+    "playsound_mapping": "open_trapdoor.copper"
+  },
```

Comment:
Would "close_door.copper" be closer to the copper trapdoor closing sound? Or are they the same on java?

Reply:
The closing and opening sounds for the trapdoor are the exact same on java and bedrock. However, `close_trapdoor.copper` does not exist, so I have to use this id.